frontend/cypress/support/backend_config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
this is empty for now, but in case that a configuration is needed the place is here